sage-android/Sage/app/src/main/java/blueprint/com/sage/admin/semester/fragments/FinishPauseSemesterFragment.java

```diff
@@ -0,0 +1,100 @@
+package blueprint.com.sage.admin.semester.fragments;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import blueprint.com.sage.R;
+import blueprint.com.sage.announcements.CreateAnnouncementActivity;
+import blueprint.com.sage.models.Semester;
+import blueprint.com.sage.models.User;
+import blueprint.com.sage.shared.interfaces.BaseInterface;
+import blueprint.com.sage.shared.views.CircleImageView;
+import blueprint.com.sage.utility.network.NetworkUtils;
+import blueprint.com.sage.utility.view.FragUtils;
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * Created by kelseylam on 8/15/16.
+ */
+public class FinishPauseSemesterFragment extends Fragment {
+
+    @Bind(R.id.pause_semester_announcement) LinearLayout mAnnouncement;
+    @Bind(R.id.announcement_user) TextView mUser;
+    @Bind(R.id.announcement_time) TextView mTime;
+    @Bind(R.id.announcement_title) TextView mTitle;
+    @Bind(R.id.announcement_body) TextView mBody;
+    @Bind(R.id.announcement_school) TextView mSchool;
+    @Bind(R.id.announcement_profile_picture) CircleImageView mPicture;
+
+    private BaseInterface mBaseInterface;
+    private Semester mSemester;
+
+    public static FinishPauseSemesterFragment newInstance(Semester semester) {
+        FinishPauseSemesterFragment fragment = new FinishPauseSemesterFragment();
+        fragment.setSemester(semester);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        mBaseInterface = (BaseInterface) getActivity();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup parent, Bundle savedInstanceState) {
+        super.onCreateView(inflater, parent, savedInstanceState);
+        View view = inflater.inflate(R.layout.fragment_finish_pause_semester, parent, false);
+        ButterKnife.bind(this, view);
+        initializeAnnouncementView();
+        return view;
+    }
+
+    private void initializeAnnouncementView() {
+        User user = mBaseInterface.getUser();
+
+        if (user != null) {
```

Comment:
you should just return if user is null to reduce indentation. Also perhaps throw an log if that happens.